gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+[versions]
+# Kotlin
+kotlin = "1.7.0"
+
+# Compose
+activityCompose = "1.5.1"
+compose = "1.2.0"
+navigationCompose = "2.5.1"
+hiltNavigationCompose = "1.0.0"
+
+# Plugins
+androidToolsPlugin = "7.3.0"
+detekt = "1.21.0"
+
+# Android libraries
+lifecycle = "2.5.1"
+workManager = "2.7.1"
+material3 = "1.0.0-alpha16"
+androidCore = "1.8.0"
+coroutines = "1.6.4"
+dagger = "2.43.2"
+dataStore = "1.0.0"
+playServicesAuth = "20.2.0"
+serialization = "1.3.3"
+
+# third party libs
+glideCompose = "1.6.1"
+timber = "5.0.1"
+
+# testing
+junit = "4.13.2"
+junitExt = "1.1.3"
+kaspresso = "1.4.1"
+
+[libraries]
+# Android libs
+androidCore = { module = "androidx.core:core-ktx", version.ref = "androidCore" }
+activityCompose = { module = "androidx.activity:activity-compose", version.ref = "activityCompose" }
+composeCompiler = { module = "androidx.compose.compiler:compiler", version.ref = "compose" }
+composeUI = { module = "androidx.compose.ui:ui", version.ref = "compose" }
+composeMaterial = { module = "androidx.compose.material:material", version.ref = "compose" }
+composeFoundation = { module = "androidx.compose.foundation:foundation", version.ref = "compose" }
+composeTooling = { module = "androidx.compose.ui:ui-tooling", version.ref = "compose" }
+vmCompose = { module = "androidx.lifecycle:lifecycle-viewmodel-compose", version.ref = "lifecycle" }
+
+dataStore = { module = "androidx.datastore:datastore-preferences", version.ref = "dataStore" }
+lifecycle = { module = "androidx.lifecycle:lifecycle-runtime-ktx", version.ref = "lifecycle" }
+playServicesAuth = { module = "com.google.android.gms:play-services-auth", version.ref = "playServicesAuth" }
+serialization = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json", version.ref = "serialization" }
+workManager = { module = "androidx.work:work-runtime-ktx", version.ref = "workManager" }
+
+coroutinesAndroid = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-android", version.ref = "coroutines" }
+coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "coroutines" }
+composeMaterial3 = { module = "androidx.compose.material3:material3", version.ref = "material3" }
+
+# third party libs
+glideComposeVersion = { module = "com.github.skydoves:landscapist-glide", version.ref = "glideCompose" }
+timber = { module = "com.jakewharton.timber:timber", version.ref = "timber" }
+detektFormatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
+
+# testing
+junit = { module = "junit:junit", version.ref = "junit" }
+junitExt = { module = "androidx.test.ext:junit", version.ref = "junitExt" }
+kaspresso = { module = "com.kaspersky.android-components:kaspresso-compose-support", version.ref = "kaspresso" }
+workManagerTest = { module = "androidx.work:work-testing", version.ref = "workManager" }
+
+# Google
```

Comment:
I've tried declaring `plugins` like this:       
```
[versions]
android = "7.3"
appPlugin = "7.3.0"
...
[plugins]
androidPlugin = { id = "android", version.ref = "android" }
appPlugin = { id = "com.android.application", version.ref = "appPlugin" }
```

Reply:
You should declare plugin in root project also with `apply false`. [Example](https://github.com/cXense/cxense-android/blob/master/build.gradle.kts#L3-L6)